sequencer/src/bin/orchestrator.rs

```diff
@@ -195,7 +195,7 @@ async fn main() {
     config.config.known_nodes_with_stake = known_nodes_with_stake;
     config.config.max_transactions = args
         .max_transactions
-        .unwrap_or(NonZeroUsize::new(2_usize.pow(MAX_NMT_DEPTH as u32)).unwrap());
+        .unwrap_or(NonZeroUsize::new(1000).unwrap()); // TODO what to set here?
```

Comment:
Something arbitrary is fine. This is going away (getting moved into the buidler) soon anyways, and what we will want to enforce instead is a maximum _size_.

Also, now that we're just using a constant instead of an expression, we can set this using `default_value` on the clap field and not bother with the `unwrap_or`